pkgs/rpm/clapper.spec

```diff
@@ -0,0 +1,64 @@
+#
+# spec file for package clapper
+#
+# Copyright (C) 2020  sp1rit
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+
+Name:				clapper
+Version:			0.0.0
+Release:			0
+Summary:			A GNOME media player
+License:			GPL-3.0
+URL:				https://github.com/Rafostar/clapper
+BuildArchitectures:		noarch
+BuildRoot:			%{_builddir}/%{name}-%{version}-build
+BuildRequires:			meson gjs
+Requires:			gjs gstreamer
+Source0:			_service
+%if 0%{?suse_version}
+Requires:			gstreamer-plugins-base gstreamer-plugins-good-gtk libgstplayer-1_0-0
+Recommends:			gstreamer-plugins-vaapi
```

Comment:
What do you think about adding `gstreamer*-libav` to "Recommends" (and "optdepends" on Arch). They are not required, but the player won't be able to handle most media files (especially videos) without it so maybe we should recommend them at least.

Reply:
If I see that right, then there is no official gstreamer1-libav for RHEL. Only RPM Fusion Free & UnitedRPMs.

I potential workaround would be to add gstreamer1-libav to my repo aswell

Reply:
Ah. Yes. You are right. I forgot Fedora ships them in rpmfusion (proprietary code). I will simply update README to tell people who wish to install/test this player from source code to install them separately.

> would be to add gstreamer1-libav to my repo aswell

The decision is up to you.

Reply:
> I potential workaround would be to add gstreamer1-libav to my repo aswell

Not gonna happen, due to legal problems with MPEG-LA. [Software which is encumbered by patent claims](https://en.opensuse.org/openSUSE:Build_Service_application_blacklist#Software_which_is_encumbered_by_patent_claims).

I think the best way is to just tell RHEL users to install gstreamer1-libav from RPM Fusion Free if they want additional codecs.